core/kazoo_call/src/kapps_call_command.erl

```diff
@@ -1519,6 +1522,44 @@ b_play(Media, Terminators, Leg, Call) ->
 b_play(Media, Terminators, Leg, Endless, Call) ->
     wait_for_noop(Call, play(Media, Terminators, Leg, Endless, Call)).
 
+%%------------------------------------------------------------------------------
+%% @doc Produces the low level AMQP request to seek through the playing media.
+%% This request will execute immediately.
+%% @end
+%%------------------------------------------------------------------------------
+-spec seek(kapps_call:call()) -> kapps_api_std_return().
+seek(Call) ->
+    seek(?DEFAULT_SEEK_DURATION, Call).
+
+-spec seek(kz_term:api_integer(), kapps_call:call()) -> kapps_api_std_return().
+seek(Duration, Call) when Duration > 0 ->
+    seek('fastforward', Duration, Call);
+seek(Duration, Call) when Duration < 0 ->
+    seek('rewind', -Duration, Call);
+seek(_Duration, _Call) ->
+    'ok'.
+
+-spec b_seek(atom(), kz_term:api_pos_integer(), kapps_call:call()) -> kapps_api_std_return().
+b_seek(Direction, Duration, Call) ->
+    wait_for_noop(Call, seek(Direction, Duration, Call)).
+
+-spec seek(atom(), kz_term:api_pos_integer(), kapps_call:call()) -> kapps_api_std_return().
+seek(_Direction, 0, _Call) ->
+    'ok';
```

Comment:
This `'ok`' is not a `kapps_api_std_return()`. Perhaps b_seek should pattern-match the 0 and return immediately rather than call `wait_for_noop` with `'ok'` in the second parameter?

Reply:
Alternatively you could return 'undefined' instead. Then the return type would be `kz_term:api_ne_binary()`

Reply:
-type kapps_api_std_return() :: kapps_api_error() |
                                {'ok', kz_json:object() |
                                 kz_term:ne_binary()
                                } |
                                'ok'.

Reply:
@jamesaimonetti I think we're a little confused on what it's not a kapps_api_std_return().  I imagine that it's because this is an OK from freeswitch?

Reply:
@jamesaimonetti turns out b_seek/3 wasn't needed, and we removed it.  Does this help?

Reply:
This appears to still return 'ok' and b_seek still uses that return value (unless I'm missing something?).